### ISSUE TITLE
Touchscreen swipes crash hyprland

### ISSUE THREAD

**Issue by UltraBlackLinux** [hyprwm/Hyprland#1952]
Hey there,

Yesterday I installed Hyprland On my PPP, but I've been running into some issues. The issue I want to highlight here is the weird touch behavior.

If I just tap on the screen, the cursor hides and seemingly moves to my tap position and clicks, as expected. Now, if I swipe on my screen, hyprland crashes, but there is nothing in the logs, not even a crash log (So I suspect it just closes? Why though?)

I would also like to configure touch gestures, but I haven't seen an option for that in the wiki. I would use `lisgd`, but not if any swipes kill hyprland.

Any idea what's happening?

**Comment by vaxerski** [comment]
checked on my laptop, no crashing occurs.

> I would also like to configure touch gestures

Not supported atm

**Comment by UltraBlackLinux** [comment]
> checked on my laptop

On the touchscreen?

**Comment by vaxerski** [comment]
ye

**Comment by UltraBlackLinux** [comment]
Did you try swiping on a window? I tried this someme more and I can only get it to crash when I swipe on windows, and even then it takes a few seconds to do.

**Comment by vaxerski** [comment]
of it crashes, send a crash report.

**Comment by UltraBlackLinux** [comment]
bruh. There is none. It's as if it closed normally. I don't know what to do...

**Comment by UltraBlackLinux** [comment]
I got an output: 
```
Thread 1 "Hyprland" received signal SIGSEGV, Segmentation fault.
0x0000aaaacaaf6fe4 in CInputManager::onTouchMove(wlr_touch_motion_event*) ()
(gdb) backtrace
#0  0x0000aaaacaaf6fe4 in CInputManager::onTouchMove(wlr_touch_motion_event*) ()
#1  0x0000ffffb44e9544 in wl_signal_emit_mutable () at /usr/lib/libwayland-server.so.0
#2  0x0000ffffb44e9544 in wl_signal_emit_mutable () at /usr/lib/libwayland-server.so.0
#3  0x0000ffffb469c9b4 in handle_touch_motion ()
    at /home/raccoon/.cache/yay/hyprland-git/src/hyprland/subprojects/wlroots/build/libwlroots.so.12032
#4  0x0000ffffb4698b70 in handle_libinput_readable ()
    at /home/raccoon/.cache/yay/hyprland-git/src/hyprland/subprojects/wlroots/build/libwlroots.so.12032
#5  0x0000ffffb44ebd40 in wl_event_loop_dispatch () at /usr/lib/libwayland-server.so.0
#6  0x0000ffffb44e8df4 in wl_display_run () at /usr/lib/libwayland-server.so.0
#7  0x0000aaaaca9fde68 in main ()
```

**Comment by vaxerski** [comment]
do you have a mouse in that? as in, a touchpad, pointer, trackball, whatever, apart from the touchscreen?

**Comment by vaxerski** [comment]
so that's why

**Comment by vaxerski** [comment]
should be fixed in 19809532df61b47750485ec7d84bedd083c809f1

**Comment by UltraBlackLinux** [comment]
Oh thank you so much that works like a charm!
